html/canvas/element/text/2d.text.measure.actualBoundingBox.whitespace.html

```diff
@@ -38,11 +38,16 @@ <h1>2d.text.measure.actualBoundingBox.whitespace</h1>
   // Values that are nominally expected to be zero might actually vary by a
   // pixel or so if the UA accounts for antialiasing at glyph edges, so we
   // allow a slight deviation.
-  _assert(Math.abs(ctx.measureText('A ').actualBoundingBoxLeft) <= 1, "Math.abs(ctx.measureText('A ').actualBoundingBoxLeft) <= 1");
-  _assert(ctx.measureText('A ').actualBoundingBoxRight >= 50, "ctx.measureText('A ').actualBoundingBoxRight >= 50");
+  var whitespaces = [0x9, 0xa, 0xc, 0xd, 0x20];
```

Comment:
Let's add at least one non-ASCII one in there (also below):
```suggestion
  var whitespaces = [0x9, 0xa, 0xc, 0xd, 0x20, 0x3000];
```